docs/requirements-docs.txt

```diff
@@ -3,6 +3,7 @@ pandas>=1.5,<2
 matplotlib>=3.6.0,<4
 nbval
 sphinx
+sphinx-rtd-theme
```

Comment:
are the two packages with similar names both requires? 
sphinx-rtd-theme
sphinx_rtd_theme

Reply:
This part I'm not exactly sure. You can see two different installation instruction:

1. https://sphinx-rtd-theme.readthedocs.io/en/stable/installing.html
2. https://pypi.org/project/sphinx-rtd-theme/

And I'm seeing bit different output than my end. So for which I just wanted to give a try with installing both.